stable/spark/Chart.yaml

```diff
@@ -1,12 +1,17 @@
 name: spark
-version: 0.2.1
+version: 1.0.0
 appVersion: 1.5.1
```

Comment:
From recent comments this is Spark `2.4.0`, so the `appVersion` should be updated to reflect that.